docs/ref/databases.txt

```diff
@@ -582,12 +625,14 @@ these methods in no-op's based in the results of such detection.
 Notes on specific fields
 ------------------------
 
-Character fields
-~~~~~~~~~~~~~~~~
+Indexes for character fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Any fields that are stored with ``VARCHAR`` column types have their
-``max_length`` restricted to 255 characters if you are using ``unique=True``
-for the field. This affects :class:`~django.db.models.CharField`,
+Any index for fields that are stored with ``VARCHAR`` column types will only
+index the 191 first characters of the fields, as index length is limited to 767
+bytes. Avoid using ``unique=True`` on fields having more than 191 characters,
+because the unicity will only be checked on the 191 first characters. This
```

Comment:
Will mysql even let you do this (unique with more than 191 on utf8mb4) without giving an error?

Reply:
You can use row_format=dynamic and innodb_file_format=barracuda to get much more than 191 chars too